lib/net_buf/buf_simple.c

```diff
@@ -73,6 +74,32 @@ void *net_buf_simple_add_mem(struct net_buf_simple *buf, const void *mem,
 	return memcpy(net_buf_simple_add(buf, len), mem, len);
 }
 
+int net_buf_simple_add_str(struct net_buf_simple *buf, const char *s)
```

Comment:
If we have a `net_buf_simple_add_str_len()`, then this could just call it with
`net_buf_simple_add_str_len(buf, str, strlen(str))`

Reply:
`strlen` would be called twice then?
- Once by `net_buf_simple_add_str()` in the wrapper over `_add_str_len()` like above
- Once by `net_buf_simple_add_str_len()` which would become a wrapper over `_add_mem(str, min(strlen(str), max_len))`.

Maybe I am missing something.

Reply:
I can do it with `net_buf_simple_add_str_len(buf, str, SIZE_MAX)` though: no limit.